---
title: " "
permalink: /PUBLICATIONS/
header:
  image: /assets/img/photos/header6.jpg
  caption: "Image credit: [Annie Cavanagh (CC BY-NC 4.0)](https://elifesciences.org/digests/51480/cancer-mutations-cell-by-cell)"
toc: true
toc_sticky: true
toc_label: " "
toc_icon: "bars"
layout: single
# classes: wide
---
# Selected publications

Lin, P.#, Yan, X., Jing, S., Wu, Y., Shan, Y., Guo, W., Gu, J., Li, Y., Zhang, H., & **Li, H.\*** (2023). Single-cell and spatially resolved transcriptomics for liver biology. *Hepatology (Baltimore, Md.)*, 10.1097/HEP.0000000000000387. Advance online publication. <a href="/assets/pdf/Single_cell_and_spatially_resolved_transcriptomics_for_liver_biology.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a> <a href="https://mp.weixin.qq.com/s/lJijLYq8vwityLLK8Ph3ew" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

Li, L.#, Cui, L.#, Lin, P.#, Liu, Z., Bao, S., Ma, X., Nan, H., Zhu, W., Cen, J., Mao, Y., Ma, X., Jiang, L., Nie, Y., Ginhoux, F., Li, Y.\*, **Li, H.\***, & Hui, L\*. (2023). Kupffer-cell-derived IL-6 is repurposed for hepatocyte dedifferentiation via activating progenitor genes from injury-specific enhancers. Cell Stem Cell. <a href="/assets/pdf/1-s2.0-S1934590923000097-main.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a> <a href="http://www.sinh.ac.cn/xwgg/kyjz/202302/t20230214_6677647.html" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a> <a href="https://mp.weixin.qq.com/s/AZETQGZwPaWNrsHOVvGwog" rel="nofollow noopener noreferrer"><i class="fa-solid fa-link" aria-hidden="true"></i>BioArt</a> <a href="https://mp.weixin.qq.com/s/RPDX-sPLKQ-O45TZIgcfHg" rel="nofollow noopener noreferrer"><i class="fa-solid fa-link" aria-hidden="true"></i>iNature</a>

Shen, B.#, Feng, F.#, Li, K., Lin, P., Ma, L., & **Li, H\***. (2022). A systematic assessment of deep learning methods for drug response prediction: from in vitro to clinical applications. Briefings in bioinformatics, bbac605. Advance online publication. <a href="/assets/pdf/bbac605.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a> <a href="https://mp.weixin.qq.com/s/tjepBBL92wY890L4SD9ahg" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

Yang J#, Lin P#, Yang M#, Liu W, Fu X, Liu D, Tao L, Huo Y, Zhang J, Hua R, Zhang Z\*, Li Y\*, Wang L\*, Xue J\*, **Li H\***, Sun Y\*. Integrated genomic and transcriptomic analysis reveals unique characteristics of hepatic metastases and pro-metastatic role of complement C1q in pancreatic ductal adenocarcinoma. Genome biology 2021, 22(1): 4. <a href="/assets/pdf/s13059-020-02222-w.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>

Feng F, Shen B, Mou X, Li Y, **Li H\***. Large-scale pharmacogenomic studies and drug response prediction for personalized cancer medicine. J Genet Genomics. 2021;48(7):540-55 <a href="/assets/pdf/1-s2.0-S1673852721000849-main.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>

Hu B#, **Li H#**, Guo W, Sun YF, Zhang X, Tang WG, Yang LX, Xu Y, Tang XY, Ding GH, Qiu SJ, Zhou J, Li YX, Fan J\*, Yang XR\*. Establishment of a hepatocellular carcinoma patient-derived xenograft platform and its application in biomarker identification. Int J Cancer 2020 Mar 15;146(6):1606-1617. <a href="/assets/pdf/Intl Journal of Cancer - 2019 - Hu - Establishment of a hepatocellular carcinoma patient‐derived xenograft platform and its.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>

Qiu Z#, **Li H#**, Zhang Z#, Zhu Z, He S, Wang X, Wang P, Qin J, Zhuang L, Wang W, Xie F, Gu Y, Zou K, Li C, Li C, Wang C, Cen J, Chen X, Shu Y, Zhang Z, Sun L, Min L, Fu Y, Huang X, Lv H, Zhou H, Ji Y, Zhang Z, Meng Z, Shi X, Zhang H\*, Li Y\*, Hui L\*. A Pharmacogenomic Landscape in Human Liver Cancers. Cancer Cell 2019 Aug 12;36(2):179-193.e11. <a href="/assets/pdf/PIIS1535610819303289.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>

Li W#, Yang L#, He Q, Hu C, Zhu L, Ma X, Ma X, Bao S, Li L, Chen Y, Deng X, Zhang X, Cen J, Zhang L, Wang Z, Xie WF, **Li H\***, Li Y\*, Hui L\*. A Homeostatic Arid1a-Dependent Permissive Chromatin State Licenses Hepatocyte Responsiveness to Liver-Injury-Associated YAP Signaling. Cell stem cell 2019 Jul 3;25(1):54-68.e5. <a href="/assets/pdf/PIIS1934590919302723.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>

Cheng J#, Wei D, Ji Y, Chen L, Yang L, Li G, Wu L, Hou T, Xie L, Ding G, **Li H\***, Li Y\*. Integrative analysis of DNA methylation and gene expression reveals hepatocellular carcinoma-specific diagnostic biomarkers. Genome Med 2018 May 30;10(1):42. <a href="/assets/pdf/s13073-018-0548-z.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>

Hu H#, **Li H#**, Jiao F, Han T, Zhuo M, Cui J, Li Y\*, Wang L\*. Association of a novel point mutation in MSH2 gene with familial multiple primary cancers. J Hematol Oncol 2017 Oct 3;10(1):158. <a href="/assets/pdf/s13045-017-0523-y.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>

# All publications

## 2024
Shen, B.#, Ma, C.#, Tang, L., Wu, Z., Peng, Z., Pan, G.\*, & **Li, H.\*** (2024). Applying machine learning for multi-individual Raman spectroscopic data to identify different stages of proliferating human hepatocytes. iScience. [https://doi.org/10.1016/j.isci.2024.109500](https://doi.org/10.1016/j.isci.2024.109500) <a href="/assets/pdf/PIIS2589004224007211.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>

## 2023

Lin, P.#, Yan, X., Jing, S., Wu, Y., Shan, Y., Guo, W., Gu, J., Li, Y., Zhang, H., & **Li, H.\*** (2023). Single-cell and spatially resolved transcriptomics for liver biology. *Hepatology (Baltimore, Md.)*, 10.1097/HEP.0000000000000387. Advance online publication. [https://doi.org/10.1097/HEP.0000000000000387](https://doi.org/10.1097/HEP.0000000000000387) <a href="/assets/pdf/Single_cell_and_spatially_resolved_transcriptomics_for_liver_biology.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a> <a href="https://mp.weixin.qq.com/s/lJijLYq8vwityLLK8Ph3ew" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

Li, L.#, Cui, L.#, Lin, P.#, Liu, Z., Bao, S., Ma, X., Nan, H., Zhu, W., Cen, J., Mao, Y., Ma, X., Jiang, L., Nie, Y., Ginhoux, F., Li, Y.\*, **Li, H.\***, & Hui, L\*. (2023). Kupffer-cell-derived IL-6 is repurposed for hepatocyte dedifferentiation via activating progenitor genes from injury-specific enhancers. Cell Stem Cell. [https://doi.org/10.1016/j.stem.2023.01.009](https://doi.org/10.1016/j.stem.2023.01.009) <a href="/assets/pdf/1-s2.0-S1934590923000097-main.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a> <a href="http://www.sinh.ac.cn/xwgg/kyjz/202302/t20230214_6677647.html" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a> <a href="https://mp.weixin.qq.com/s/AZETQGZwPaWNrsHOVvGwog" rel="nofollow noopener noreferrer"><i class="fa-solid fa-link" aria-hidden="true"></i>BioArt</a> <a href="https://mp.weixin.qq.com/s/RPDX-sPLKQ-O45TZIgcfHg" rel="nofollow noopener noreferrer"><i class="fa-solid fa-link" aria-hidden="true"></i>iNature</a>

## 2022

Shen, B.#, Feng, F.#, Li, K., Lin, P., Ma, L., & **Li, H\***. (2022). A systematic assessment of deep learning methods for drug response prediction: from in vitro to clinical applications. *Briefings in bioinformatics*, bbac605. Advance online publication. [https://doi.org/10.1093/bib/bbac605](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbac605/6961794?utm_source=authortollfreelink&utm_campaign=bib&utm_medium=email&guestAccessKey=b7e964fc-aafa-438c-8414-a8bdc22e2313) <a href="/assets/pdf/bbac605.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a> <a href="https://mp.weixin.qq.com/s/tjepBBL92wY890L4SD9ahg" rel="nofollow noopener noreferrer"><i class="fa-solid fa-arrow-up-right-from-square" aria-hidden="true"></i></a>

Liu D#, **Li H#**, Dong H#, Qu M, Yang L, Chen L, Li Y, Wang H*, He Y*. Spatial Multiomics Analysis Reveals Only Minor Genetic and Epigenetic Changes in Human Liver Cancer Stem-Like Cells Compared With Other Tumor Parenchymal Cells. Front Cell Dev Biol. 2022 Feb 9;10:810687. [doi: 10.3389/fcell.2022.810687](doi: 10.3389/fcell.2022.810687). PMID: 35223840; PMCID: PMC8863946.

## 2021

Chen, Y., Zhou, C., Li, H., **Li, H.\***, & Li, Y\*. (2021). Identifying Key Genes for Nasopharyngeal Carcinoma by Prioritized Consensus Differentially Expressed Genes Caused by Aberrant Methylation. J Cancer, 12(3), 874-884. [https://doi.org/10.7150/jca.49392](https://doi.org/10.7150/jca.49392) 

Yang, J.#, Lin, P.#, Yang, M.#, Liu, W., Fu, X., Liu, D., Tao, L., Huo, Y., Zhang, J., Hua, R., Zhang, Z.\*, Li, Y.\*, Wang, L.\*, Xue, J.\*, **Li, H.\***, & Sun, Y.\* (2021). Integrated genomic and transcriptomic analysis reveals unique characteristics of hepatic metastases and pro-metastatic role of complement C1q in pancreatic ductal adenocarcinoma. Genome Biol, 22(1), 4. [https://doi.org/10.1186/s13059-020-02222-w](https://doi.org/10.1186/s13059-020-02222-w) 

Chen, Y., Wang, Z., **Li, H.\***, & Li, Y.\* (2021). Integrative Analysis Identified a 6-miRNA Prognostic Signature in Nasopharyngeal Carcinoma. Front Cell Dev Biol, 9, 661105. [https://doi.org/10.3389/fcell.2021.661105](https://doi.org/10.3389/fcell.2021.661105) 

Feng, F., Shen, B., Mou, X., Li, Y., & **Li, H\***. (2021). Large-scale pharmacogenomic studies and drug response prediction for personalized cancer medicine. J Genet Genomics, 48(7), 540-551. [https://doi.org/10.1016/j.jgg.2021.03.007](https://doi.org/10.1016/j.jgg.2021.03.007)

## 2020

Yang, L., Zhang, Z., Sun, Y., Pang, S., Yao, Q., Lin, P., Cheng, J., Li, J., Ding, G., Hui, L., Li, Y.\*, & **Li, H.\*** (2020). Integrative analysis reveals novel driver genes and molecular subclasses of hepatocellular carcinoma. Aging (Albany NY), 12. [https://doi.org/10.18632/aging.104047](https://doi.org/10.18632/aging.104047)

## 2019

Li, W.#, Yang, L.#, He, Q., Hu, C., Zhu, L., Ma, X., Ma, X., Bao, S., Li, L., Chen, Y., Deng, X., Zhang, X., Cen, J., Zhang, L., Wang, Z., Xie, W. F., **Li, H.\***, Li, Y.\*, & Hui, L.\* (2019). A Homeostatic Arid1a-Dependent Permissive Chromatin State Licenses Hepatocyte Responsiveness to Liver-Injury-Associated YAP Signaling. Cell Stem Cell, 25(1), 54-68 e55. [https://doi.org/10.1016/j.stem.2019.06.008](https://doi.org/10.1016/j.stem.2019.06.008) 

Sun, L., Wang, Y., Cen, J., Ma, X., Cui, L., Qiu, Z., Zhang, Z., **Li, H.**, Yang, R. Z., Wang, C., Chen, X., Wang, L., Ye, Y., Zhang, H., Pan, G., Kang, J. S., Ji, Y., Zheng, Y. W., Zheng, S., & Hui, L. (2019). Modelling liver cancer initiation with organoids derived from directly reprogrammed human hepatocytes. Nat Cell Biol. [https://doi.org/10.1038/s41556-019-0359-5](https://doi.org/10.1038/s41556-019-0359-5) 

Hu, B.#, **Li, H.#**, Guo, W., Sun, Y. F., Zhang, X., Tang, W. G., Yang, L. X., Xu, Y., Tang, X. Y., Ding, G. H., Qiu, S. J., Zhou, J., Li, Y. X., Fan, J.\*, & Yang, X. R.\*(2019). Establishment of a hepatocellular carcinoma patient-derived xenograft platform and its application in biomarker identification. Int J Cancer, 146(6), 1606-1617. [https://doi.org/10.1002/ijc.32564](https://doi.org/10.1002/ijc.32564) 

Chen, X. M., Guo, K., **Li, H.**, Lu, Q. F., Yang, C., Yu, Y., Hou, J. W., Fei, Y. D., Sun, J., Wang, J., Li, Y. X., & Li, Y. G. (2019). A novel mutation KCNQ1p.Thr312del is responsible for long QT syndrome type 1. Heart Vessels, 34(1), 177-188. [https://doi.org/10.1007/s00380-018-1223-4](https://doi.org/10.1007/s00380-018-1223-4) 

Qiu, Z.#, **Li, H.#**, Zhang, Z., Zhu, Z., He, S., Wang, X., Wang, P., Qin, J., Zhuang, L., Wang, W., Xie, F., Gu, Y., Zou, K., Li, C., Li, C., Wang, C., Cen, J., Chen, X., Shu, Y., Zhang, Z., Sun, L., Min, L., Fu, Y., Huang, X., Lv, H., Zhou, H., Ji, Y., Zhang, Z., Meng, Z., Shi, X., Zhang, H.\*, Li, Y.\*, & Hui, L.\* (2019). A Pharmacogenomic Landscape in Human Liver Cancers. Cancer Cell, 36(2), 179-193 e111. [https://doi.org/10.1016/j.ccell.2019.07.001](https://doi.org/10.1016/j.ccell.2019.07.001) 

## 2018

Cheng, J.#, Wei, D., Ji, Y., Chen, L., Yang, L., Li, G., Wu, L., Hou, T., Xie, L., Ding, G.\*, **Li, H.\***, & Li, Y.\* (2018). Integrative analysis of DNA methylation and gene expression reveals hepatocellular carcinoma-specific diagnostic biomarkers. Genome Med, 10(1), 42. [https://doi.org/10.1186/s13073-018-0548-z](https://doi.org/10.1186/s13073-018-0548-z) 

Li, J.#, Yao, Q., Feng, F., He, S., Lin, P., Yang, L., Yang, C., **Li, H.\***, & Li, Y.\* (2018). Systematic identification of rabbit LncRNAs reveals functional roles in atherosclerosis. Biochim Biophys Acta, 1864(6 Pt B), 2266-2273. [https://doi.org/10.1016/j.bbadis.2017.12.040](https://doi.org/10.1016/j.bbadis.2017.12.040) <a href="/assets/pdf/1-s2.0-S0925443918300012-main.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>

He, S.#, Hu, B.#, Li, C., Lin, P., Tang, W. G., Sun, Y. F., Feng, F. Y., Guo, W., Li, J., Xu, Y., Yao, Q. L., Zhang, X., Qiu, S. J., Zhou, J., Fan, J., Li, Y. X., **Li, H.\***, & Yang, X. R.\* (2018). PDXliver: a database of liver cancer patient derived xenograft mouse models. BMC Cancer, 18(1), 550. [https://doi.org/10.1186/s12885-018-4459-6](https://doi.org/10.1186/s12885-018-4459-6) 

Wu, L.#, Yao, Q., Lin, P., Li, Y.\*, & **Li, H.\*** (2018). Comparative transcriptomics reveals specific responding genes associated with atherosclerosis in rabbit and mouse models. PloS one, 13(8), e0201618. [https://doi.org/10.1371/journal.pone.0201618](https://doi.org/10.1371/journal.pone.0201618) 

Xiao, Q.#, Sun, Y., Dobi, A., Srivastava, S., Wang, W., Srivastava, S., Ji, Y., Hou, J., Zhao, G. P., Li, Y.\*, & **Li, H.\*** (2018). Systematic analysis reveals molecular characteristics of ERG-negative prostate cancer. Sci Rep, 8(1), 12868. [https://doi.org/10.1038/s41598-018-30325-9](https://doi.org/10.1038/s41598-018-30325-9) <a href="/assets/pdf/s41598-018-30325-9.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>

Chao, M. J., Kim, K. H., Shin, J. W., Lucente, D., Wheeler, V. C., **Li, H.,** Roach, J. C., Hood, L., Wexler, N. S., Jardim, L. B., Holmans, P., Jones, L., Orth, M., Kwak, S., MacDonald, M. E., Gusella, J. F., & Lee, J. M. (2018). Population-specific genetic modification of Huntington's disease in Venezuela. PLoS Genet, 14(5), e1007274. [https://doi.org/10.1371/journal.pgen.1007274](https://doi.org/10.1371/journal.pgen.1007274)

## Before 2018

Yao, Q.#, Wu, L., Li, J., Yang, L. G., Sun, Y., Li, Z., He, S., Feng, F., **Li, H.\***, & Li, Y.\* (2017). Global Prioritizing Disease Candidate lncRNAs via a Multi-level Composite Network. Sci Rep, 7, 39516. [https://doi.org/10.1038/srep39516](https://doi.org/10.1038/srep39516) 

Wu, L.#, Pang, S., Yao, Q., Jian, C., Lin, P., Feng, F., **Li, H.\***, & Li, Y.\* (2017). Population-based study of effectiveness of neoadjuvant radiotherapy on survival in US rectal cancer patients according to age. Sci Rep, 7(1), 3471. [https://doi.org/10.1038/s41598-017-02992-7](https://doi.org/10.1038/s41598-017-02992-7) 

Hu, H.#, **Li, H.#**, Jiao, F., Han, T., Zhuo, M., Cui, J., Li, Y., & Wang, L.\* (2017). Association of a novel point mutation in MSH2 gene with familial multiple primary cancers. J Hematol Oncol, 10(1), 158. [https://doi.org/10.1186/s13045-017-0523-y](https://doi.org/10.1186/s13045-017-0523-y)

Chen, S.#, Zhou, K., Yang, L., Ding, G., & **Li, H.\*** (2017). Racial Differences in Esophageal Squamous Cell Carcinoma: Incidence and Molecular Features. Biomed Res Int, 2017, 1204082. [https://doi.org/10.1155/2017/1204082](https://doi.org/10.1155/2017/1204082) 

Luo, H.#, **Li, H.#**, Hu, Z., Wu, H., Liu, C., Li, Y., Zhang, X., Lin, P., Hou, Q., Ding, G., Wang, Y., Li, S., Wei, D., Qiu, F., Li, Y.\*, & Wu, S.\* (2016). Noninvasive diagnosis and monitoring of mutations by deep sequencing of circulating tumor DNA in esophageal squamous cell carcinoma. Biochem Biophys Res Commun, 471(4), 596-602. [https://doi.org/10.1016/j.bbrc.2016.02.011](https://doi.org/10.1016/j.bbrc.2016.02.011) 

Wang, Z.#, Zhang, J.#, **Li, H.#**, Li, J., Niimi, M., Ding, G., Chen, H., Xu, J., Zhang, H., Xu, Z., Dai, Y., Gui, T., Li, S., Liu, Z., Wu, S., Cao, M., Zhou, L., Lu, X., Wang, J., Yang, J., Fu, Y., Yang, D., Song, J., Zhu, T., Li, S., Ning, B., Wang, Z., Koike, T., Shiomi, M., Liu, E., Chen, L., Fan, J., Chen, Y. E.\*, & Li, Y.\* (2016). Hyperlipidemia-associated gene variations and expression patterns revealed by whole-genome and transcriptome sequencing of rabbit models. Sci Rep, 6, 26942. [https://doi.org/10.1038/srep26942](https://doi.org/10.1038/srep26942) 

Xu, S.#, Zhu, X.#, **Li, H.#**, Hu, Y., Zhou, J., He, D., Feng, Y., Lu, L., Du, G., Hu, Y., Liu, T., Wang, Z., Ding, G., Chen, J., Gao, S., Wu, F., Xue, Z., Li, Y.\*, & Fan, G.\* (2016). The 14th Ile residue is essential for Leptin function in regulating energy homeostasis in rat. Sci Rep, 6, 28508. [https://doi.org/10.1038/srep28508](https://doi.org/10.1038/srep28508) 

Qiu, Z., Zou, K., Zhuang, L., Qin, J., **Li, H.**, Li, C., Zhang, Z., Chen, X., Cen, J., Meng, Z., Zhang, H., Li, Y., & Hui, L. (2016). Hepatocellular carcinoma cell lines retain the genomic and transcriptomic landscapes of primary human cancers. Sci Rep, 6, 27411. [https://doi.org/10.1038/srep27411](https://doi.org/10.1038/srep27411) 

Liu, Z., Gui, T., Wang, Z., **Li, H.**, Fu, Y., Dong, X., & Li, Y. (2016). cisASE: a likelihood-based method for detecting putative cis-regulated allele-specific expression in RNA sequencing data. Bioinformatics, 32(21), 3291-3297. [https://doi.org/10.1093/bioinformatics/btw416](https://doi.org/10.1093/bioinformatics/btw416) 

**Li, H.#**, Zhu, Y., Tang, X., Li, J., Li, Y., Zhong, Z., Ding, G., & Li, Y.\* (2015). Integrated analysis of transcriptome in cancer patient-derived xenografts. PloS one, 10(5), e0124780. [https://doi.org/10.1371/journal.pone.0124780](https://doi.org/10.1371/journal.pone.0124780) 

Liu, T.#, Yu, L., Ding, G., Wang, Z., Liu, L., **Li, H.\***, & Li, Y.\* (2015). Gene Coexpression and Evolutionary Conservation Analysis of the Human Preimplantation Embryos. Biomed Res Int, 2015, 316735. [https://doi.org/10.1155/2015/316735](https://doi.org/10.1155/2015/316735) 

Liu, T.#, Yu, L., Liu, L., **Li, H.\***, & Li, Y.\* (2015). Comparative Transcriptomes and EVO-DEVO Studies Depending on Next Generation Sequencing. Comput Math Methods Med, 2015, 896176. [https://doi.org/10.1155/2015/896176](https://doi.org/10.1155/2015/896176) 

Liu, T.#, **Li, H.#**, Ding, G., Wang, Z., Chen, Y., Liu, L., Li, Y., & Li, Y\*. (2015). Comparative Genome of GK and Wistar Rats Reveals Genetic Basis of Type 2 Diabetes. PloS one, 10(11), e0141859. [https://doi.org/10.1371/journal.pone.0141859](https://doi.org/10.1371/journal.pone.0141859) 

Ameziane, N., May, P., Haitjema, A., van de Vrugt, H. J., van Rossum-Fikkert, S. E., Ristic, D., Williams, G. J., Balk, J., Rockx, D., **Li, H.**, Rooimans, M. A., Oostra, A. B., Velleuer, E., Dietrich, R., Bleijerveld, O. B., Maarten Altelaar, A. F., Meijers-Heijboer, H., Joenje, H., Glusman, G., Roach, J., Hood, L., Galas, D., Wyman, C., Balling, R., den Dunnen, J., de Winter, J. P., Kanaar, R., Gelinas, R., & Dorsman, J. C. (2015). A novel Fanconi anaemia subtype associated with a dominant-negative mutation in RAD51. Nat Commun, 6, 8829. [https://doi.org/10.1038/ncomms9829](https://doi.org/10.1038/ncomms9829) 

He, Y., Zeng, K., Zhang, X., Chen, Q., Wu, J., **Li, H.**, Zhou, Y., Glusman, G., Roach, J., Etheridge, A., Qing, S., Tian, Q., Lee, I., Tian, X., Wang, X., Wu, Z., Hood, L., Ding, Y., & Wang, K. (2015). A gain-of-function mutation in TRPV3 causes focal palmoplantar keratoderma in a Chinese family. J Invest Dermatol, 135(3), 907-909. [https://doi.org/10.1038/jid.2014.429](https://doi.org/10.1038/jid.2014.429) 

Zhang, M., **Li, H.**, He, Y., Sun, H., Xia, L., Wang, L., Sun, B., Ma, L., Zhang, G., Li, J., Li, Y., & Xie, L. (2015). Construction and Deciphering of Human Phosphorylation-Mediated Signaling Transduction Networks. J Proteome Res, 14(7), 2745-2757. [https://doi.org/10.1021/acs.jproteome.5b00249](https://doi.org/10.1021/acs.jproteome.5b00249)

Sun, B., Zhang, M., Cui, P., **Li, H.**, Jia, J., Li, Y., & Xie, L. (2015). Nonsynonymous Single-Nucleotide Variations on Some Posttranslational Modifications of Human Proteins and the Association with Diseases. Comput Math Methods Med, 2015, 124630. [https://doi.org/10.1155/2015/124630](https://doi.org/10.1155/2015/124630) 

Lee, J. M., Kim, K. H., Shin, A., Chao, M. J., Abu Elneel, K., Gillis, T., Mysore, J. S., Kaye, J. A., Zahed, H., Kratter, I. H., Daub, A. C., Finkbeiner, S., **Li, H.**, Roach, J. C., Goodman, N., Hood, L., Myers, R. H., MacDonald, M. E., & Gusella, J. F. (2015). Sequence-Level Analysis of the Major European Huntington Disease Haplotype. Am J Hum Genet, 97(3), 435-444. [https://doi.org/10.1016/j.ajhg.2015.07.017](https://doi.org/10.1016/j.ajhg.2015.07.017) 

Li, J., Jia, J., **Li, H.**, Yu, J., Sun, H., He, Y., Lv, D., Yang, X., Glocker, M. O., & Ma, L. (2014). SysPTM 2.0: an updated systematic resource for post-translational modification. Database: the journal of biological databases and curation, 2014. 

Hu, H., Roach, J. C., Coon, H., Guthery, S. L., Voelkerding, K. V., Margraf, R. L., Durtschi, J. D., Tavtigian, S. V., Wu, W., Scheet, P., & others. (2014). A unified test of linkage analysis and rare-variant association for analysis of pedigree sequence data. Nature biotechnology. 

**Li, H.#**, Glusman, G., Huff, C., Caballero, J.\*, & Roach, J. C.\* (2014). Accurate and Robust Prediction of Genetic Relationship from Whole-Genome Sequences. PloS one, 9, e85437. 

**H. Li#**, G. Glusman, H. Hu, J. Caballero, R. Hubley, D. Witherspoon, S.L. Guthery, D.E. Mauldin, L.B. Jorde, L. Hood, J.C. Roach\*, C.D. Huff\*, Relationship Estimation from Whole-Genome Sequence Data, PLoS genetics, 10 (2014) e1004144. <a href="/assets/pdf/Li-2014-Relationship-estimation-from-whole-.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>


Stittrich, A. B., Lehman, A., Bodian, D. L., Ashworth, J., Zong, Z., **Li, H.**, Lam, P., Khromykh, A., Iyer, R. K., Vockley, J. G., Baveja, R., Silva, E. S., Dixon, J., Leon, E. L., Solomon, B. D., Glusman, G., Niederhuber, J. E., Roach, J. C., & Patel, M. S. (2014). Mutations in NOTCH1 cause Adams-Oliver syndrome. Am J Hum Genet, 95(3), 275-284. [https://doi.org/10.1016/j.ajhg.2014.07.011](https://doi.org/10.1016/j.ajhg.2014.07.011) 

Lehman, A., Stittrich, A. B., Glusman, G., Zong, Z., **Li, H.**, Eydoux, P., Senger, C., Lyons, C., Roach, J. C., & Patel, M. (2014). Diffuse angiopathy in Adams-Oliver syndrome associated with truncating DOCK6 mutations. Am J Med Genet A, 164A(10), 2656-2662. [https://doi.org/10.1002/ajmg.a.36685](https://doi.org/10.1002/ajmg.a.36685) 

Gierman, H. J., Fortney, K., Roach, J. C., Coles, N. S., **Li, H.**, Glusman, G., Markov, G. J., Smith, J. D., Hood, L., Coles, L. S., & Kim, S. K. (2014). Whole-genome sequencing of the world's oldest people. PloS one, 9(11), e112430. [https://doi.org/10.1371/journal.pone.0112430](https://doi.org/10.1371/journal.pone.0112430) 

Gou, X., Wang, Z., Li, N., Qiu, F., Xu, Z., Yan, D., Yang, S., Jia, J., Kong, X., Wei, Z., Lu, S., Lian, L., Wu, C., Wang, X., Li, G., Ma, T., Jiang, Q., Zhao, X., Yang, J., Liu, B., Wei, D., **Li, H.**, Yang, J., Yan, Y., Zhao, G., Dong, X., Li, M., Deng, W., Leng, J., Wei, C., Wang, C., Mao, H., Zhang, H., Ding, G., & Li, Y. (2014). Whole-genome sequencing of six dog breeds from continuous altitudes reveals adaptation to high-altitude hypoxia. Genome Res, 24(8), 1308-1315. [https://doi.org/10.1101/gr.171876.113](https://doi.org/10.1101/gr.171876.113) 

Walters, K.-A., Olsufka, R., Kuestner, R. E., Cho, J. H., **Li, H.**, Zornetzer, G. A., Wang, K., Skerrett, S. J., & Ozinsky, A. (2013). Francisella tularensis subsp. tularensis induces a unique pulmonary inflammatory response: role of bacterial gene expression in temporal regulation of host defense responses. PloS one, 8, e62412. 

Wang, K., Yuan, Y., **Li, H.**, Cho, J.-H., Huang, D., Gray, L., Qin, S., & Galas, D. J. (2013). The spectrum of circulating RNA: a window into systems toxicology. toxicological sciences, kft014. 

Hecker, M., Lorenz, P., Steinbeck, F., **Hong, L.**, Riemekasten, G., Li, Y., Zettl, U. K., & Thiesen, H.-J. (2012). Computational analysis of high-density peptide microarray data with application from systemic sclerosis to multiple sclerosis. Autoimmunity reviews, 11, 180–190. 

Wang, K., **Li, H.**, Yuan, Y., Etheridge, A., Zhou, Y., Huang, D., Wilmes, P., & Galas, D. (2012). The complex exogenous RNA spectra in human plasma: an interface with human gut biota? PloS one, 7, e51009. 

Liu, Y.-S., Luo, X.-Y., Li, Q.-R., **Li, H.**, Li, C., Ni, H., Li, R.-X., Wang, R., Hu, H.-c., Pan, Y.-j., & others. (2012). Shotgun and targeted proteomics reveal that pre-surgery serum levels of LRG1, SAA, and C4BP may refine prognosis of resected squamous cell lung cancer. Journal of molecular cell biology, 4, 344–347. 

He, Y., Zhang, M., Ju, Y., Yu, Z., Lv, D., Sun, H., Yuan, W., He, F., Zhang, J., **Li, H.**, & others. (2012). dbDEPC 2.0: updated database of differentially expressed proteins in human cancers. Nucleic acids research, 40, D964–D971. 

Wang, Z., Ding, G., Geistlinger, L., **Li, H.**, Liu, L., Zeng, R., Tateno, Y., & Li, Y. (2011). Evolution of protein phosphorylation for distinct functional modules in vertebrate genomes. Molecular biology and evolution, 28, 1131-1140. [https://doi.org/10.1093/molbev/msq268](https://doi.org/10.1093/molbev/msq268) 

Zhao, S., Xu, W., Jiang, W., Yu, W., Lin, Y., Zhang, T., Yao, J., Zhou, L., Zeng, Y., **Li, H.**, & others. (2010). Regulation of cellular metabolism by protein lysine acetylation. Science, 327, 1000–1004. 

Wang, Q., Zhang, Y., Yang, C., Xiong, H., Lin, Y., Yao, J., **Li, H.**, Xie, L., Zhao, W., Yao, Y., & others. (2010). Acetylation of metabolic enzymes coordinates carbon source utilization and metabolic flux. Science, 327, 1004–1007. 

**Li, H.#**, He, Y., Ding, G., Wang, C., Xie, L.\*, & Li, Y.\* (2010). dbDEPC: a database of differentially expressed proteins in human cancers. Nucleic acids research, 38, D658-664. [https://doi.org/10.1093/nar/gkp933](https://doi.org/10.1093/nar/gkp933) <a href="/assets/pdf/gkp933.pdf" rel="nofollow noopener noreferrer"><i class="fa-regular fa-file-pdf" aria-hidden="true"></i></a>

Li, S.-J.#, Peng, M.#, **Li, H.#**, Liu, B.-S., Wang, C., Wu, J.-R., Li, Y.-X.\*, & Zeng, R.\* (2009). Sys-BodyFluid: a systematical database for human body fluid proteome research. Nucleic acids research, 37, D907–D912. 

Li, S.#, **Li, H.#**, Li, M., Shyr, Y., Xie, L.\*, & Li, Y.\* (2009). Improved prediction of lysine acetylation by support vector machines. Protein and peptide letters, 16, 977–983. 

Zheng, G.#, **Li, H.#**, Wang, C., Sheng, Q., Fan, H., Yang, S., Liu, B., Dai, J., Zeng, R., & Xie, L.\* (2009). A platform to standardize, store, and visualize proteomics experimental data. Acta biochimica et biophysica Sinica, 41, 273–279. 

**Li, H.#**, Xing, X.#, Ding, G., Li, Q., Wang, C., Xie, L., Zeng, R.\*, & Li, Y.\* (2009). SysPTM: a systematic resource for proteomic research on post-translational modifications. Molecular & cellular proteomics: MCP, 8, 1839-1849. [https://doi.org/10.1074/mcp.M900030-MCP200](https://doi.org/10.1074/mcp.M900030-MCP200) 

**Li, H.#**, Ding, G., Xie, L.\*, & Li, Y.\* (2009). PAnnBuilder: an R package for assembling proteomic annotation data. Bioinformatics (Oxford, England), 25, 1094-1095. [https://doi.org/10.1093/bioinformatics/btp100](https://doi.org/10.1093/bioinformatics/btp100) 

Ding, G., Sun, Y., **Li, H.**, Wang, Z., Fan, H., Wang, C., Yang, D., & Li, Y. (2008). EPGD: a comprehensive web resource for integrating and displaying eukaryotic paralog/paralogon information. Nucleic acids research, 36, D255-262. [https://doi.org/10.1093/nar/gkm924](https://doi.org/10.1093/nar/gkm924) 


